buildscripts/run_arm64_tests_in_docker.sh

```diff
@@ -10,15 +10,23 @@ else
   DOCKER_ARGS=
 fi
 
+
+cat <<EOF >> "${grpc_java_dir}/gradle.properties"
+skipAndroid=true
+skipCodegen=true
+org.gradle.jvmargs=-Xms128m -Xmx1024m
```

Comment:
Can you remove the `-Xms` setting? It is probably not needed.

Reply:
This works, build: https://source.cloud.google.com/results/invocations/3397493e-fa81-4ae0-8190-fedc5e7d2823/targets